table.js

```diff
@@ -0,0 +1,15 @@
+function ShowHide(columnClass) {
+  let columnData = document.querySelectorAll("." + columnClass);
+  CheckIfShowOrHide(columnData);
+}
+
+function CheckIfShowOrHide(columnData) {
```

Comment:
This function does not only check if the elements should be hidden or not it is also hiding them. Please use a more speaking name.